core/src/test/resources/HistoryServerExpectations/complete_stage_list_json_expectation.json

```diff
@@ -6,6 +6,7 @@
   "numCompleteTasks" : 8,
   "numFailedTasks" : 0,
   "executorRunTime" : 162,
+  "executorCpuTime" : 0,
```

Comment:
Oh no, these are expected outputs. I think the inputs are stored under `src/test/resources/spark-events`.